rpcserver.go

```diff
@@ -2048,10 +2048,36 @@ func (r *rpcServer) FundVirtualPsbt(ctx context.Context,
 		}
 
 	case req.GetRaw() != nil:
-		raw := req.GetRaw()
-		if len(raw.Inputs) > 0 {
-			return nil, fmt.Errorf("template inputs not yet " +
-				"supported")
+		var (
+			raw     = req.GetRaw()
+			prevIDs = []asset.PrevID{}
```

Comment:
nit: to avoid allocating a zero-length slice, can just be `prevIDs []asset.PrevID` (so without the equal sign, just a variable declaration that ends up being `nil`, which is fine for `append()`).